components/overlay/index.css

```
@@ -10,15 +10,17 @@ governing permissions and limitations under the License.
 */
 
 %spectrum-overlay {
-  --spectrum-overlay-animation-distance: var(--spectrum-picker-m-texticon-popover-offset-y);
+  --spectrum-animation-distance: 6px;
+  --spectrum-overlay-animation-distance: var(--spectrum-animation-distance);
```

```suggestion
  --spectrum-overlay-animation-distance: 6px;
```
Is there a reason to have `--spectrum-animation-distance` here which is named like a global token? Otherwise I think it can be removed.